stories/webapp/accountdashboard/WordPressIntegrationStory.js

```diff
@@ -0,0 +1,105 @@
+import React, { Fragment } from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { withModalTrigger, Modal, Button, InputWithButton, Icon } from 'lib';
+
+const stories = storiesOf('Web app', module);
+
+const ModalTrigger = withModalTrigger({
+  children: <Icon name="integration" />,
+  types: ['icon-only']
+});
+
+stories.add('WordPressIntegrationStory', () => {
+  const modalSubHeader = (
+    <Fragment>
+      <Icon name="integration" className="m-quarter mr-half" />
+      Wordpress v5.4
+      <span className="text-xs text-green-primary bg-green-90 px-quarter py-tenth inline-flex rounded-sm font-semi-bold mx-quarter">
+        CONNECTED
+      </span>
+      <span className="text-neutral-primary absolute right-0 mr-base">
+        Expires on 30/04/2020
+      </span>
+    </Fragment>
+  );
+
+  return (
+    <ModalTrigger>
+      <Modal.Container>
+        <Modal.Header text={modalSubHeader}>WordPress demo</Modal.Header>
+        <Modal.Body className="has-columns">
+          <div className="modal__column bg-neutral-98">
+            <div className="border border-solid border-neutral-90 rounded bg-white mb-half p-half flex items-center justify-between">
+              <label // eslint-disable-line jsx-a11y/label-has-associated-control
```

Comment:
💚 How annoying you had to add this! I guess that's the problem with eslint thought, it looks at one file for issues, and doesn't know that `inputId` is being passed down to an `<input>` element further down the chain!

I wonder if changing the name of the `<InputWithButton />` prop from `inputId` => `id` would fix this? Up to you if you wan't to follow this line of enquiry, as it is out of scope of this PR, and would lead to follow up work in the App!

You can always disable a rule for the whole file with `/* eslint-disable jsx-a11y/label-has-associated-control */`, but again, optional as this currently works 👍